web/traction.js

```diff
@@ -1,62 +1,53 @@
 var axios = require('axios')
 var fs = require('fs')
 
-const fetchAllEvents = (repo, totalStars) => {
+const fetchAllEvents = async (repo, totalStars) => {
   const MAX_PER_PAGE = 100
-  const baseUrl =
-    'https://api.github.com/repos/supabase/' + repo + '/stargazers?per_page=' + MAX_PER_PAGE
+  const baseUrl = `https://api.github.com/repos/supabase/${repo}/stargazers?per_page=${MAX_PER_PAGE}`
 
   //Start fetching every page of repos.
   const fetchPromises = [],
     pageCount = Math.ceil(totalStars / MAX_PER_PAGE)
   for (let pageI = 1; pageI <= pageCount; ++pageI) {
-    const fetchPagePromise = axios.get(baseUrl + '&page=' + pageI, {
-      headers: { Accept: 'application/vnd.github.v3.star+json' },
+    const fetchPagePromise = axios.get(`${baseUrl}&page=${pageI}`, {
+      headers: {
+        Accept: 'application/vnd.github.v3.star+json',
+      },
```

Comment:
@soedirgo does this require an API key?

Reply:
I _think_ we don't need it yet (API requests per run was about 44, hourly limit is 60). It may fail if there's >1 build in an hour, but should at least build once per day. Maybe we should leave it be for now and see if it works?

Reply:
OK I'll merge and let's see! I may block our builds/deploys on netlify though. Mind starting an issue to track?